### ISSUE TITLE
vs 2017 support

### ISSUE THREAD

**Issue by furesoft** [JetBrains/Nitra#51]
pls vs 2017 support

**Comment by jianyunli** [comment]
Yes, it seems the documentation is a little outdated. Can the author provide a step by step tutorial for beginners for VS 2017?

**Comment by nekomatic** [comment]
VS 2017 support pls, no Nitra projects under Nemerle.
